O/OpenBLAS/OpenBLASHighCoreCount@0.3.13/build_tarballs.jl

```diff
@@ -0,0 +1,17 @@
+using BinaryBuilder
+
+include("../common.jl")
+
+# Collection of sources required to build OpenBLAS
+name = "OpenBLASHighCoreCount"
+version = v"0.3.13"
+
+sources = openblas_sources(version)
+script = openblas_script(num_64bit_threads=128)
```

Comment:
Should this number be bumped higher?

@chriselrod thoughts?

Reply:
The Ampere Altra has 80 cores, and you can get [two sockets per motherboard](https://www.anandtech.com/show/16315/the-ampere-altra-review), but for x64 I think dual socket Epyc (for 128 cores) is the maximum.

What's the cost of increasing the maximum number of threads?

Reply:
> What's the cost of increasing the maximum number of threads?

As far as I understand, bore memory allocated by OpenBLAS